sql/select_queries.sql

```diff
@@ -0,0 +1,30 @@
+/* select prices list for each cinema */
+use cinema_sessions;
+SELECT DISTINCT c.title, c.building_address, t.price FROM movie_sessions s INNER JOIN cinemas c ON s.cinema_id = c.id INNER JOIN tickets t ON t.id = s.ticket_id;
```

Comment:
I do not see Create statements and why do you need "use cinema_sessions;" ?

Reply:
Create statements are on script.sql file, in select_queries are only select statements. Use statement is used to select a database and to avoid making some query into default database.

Reply:
Why are they in BIN format?

Reply:
how can i review it ? please add this files not in BIN format

Reply:
It is available if open directly in branch  task-sql, but I wil try to fix it in an hour if required.

Reply:
Fixed, sorry for taking so long time, had urgent matters.